### ISSUE TITLE
Notification View does not work correctly

### ISSUE THREAD

**Issue by rafsanulhasan** [aalok05/CodeHub#106]
## I'm submitting a...

```
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  
[ ] Feature request
[ ] Question
```

## Current behavior
Clicking on a notification from the notification list shows **Readme.md** of that repo

## Expected behavior
It should show a specific issue or PR

## Steps to reproduce the behavior
Whenever new notification(s) arrive, click on the notifications icon and then click on any notification from the list.

## Environment
Windows 10 Fall update

**Comment by rafsanulhasan** [comment]
Fixed in #108

**Comment by aalok05** [comment]
Merged.
